lib/Services/Bridge.php

```diff
@@ -53,6 +53,16 @@ public static function isMPSendingServiceEnabled() {
     }
   }
 
+  public function isMailpoetSendingServiceEnabled() {
```

Comment:
Is it on purpose that we keep also the `isMPSendingServiceEnabled` static method?

Reply:
The original static method is still used in many places. I should probably mark it as `@deprecated`.